src/storage/filesystem.py

```diff
@@ -12,30 +11,34 @@
 from storage import InventoryStorage, InventoryItem
 
 
-class FilesystemInventory(InventoryStorage):    # pylint: disable=too-many-ancestors, abstract-method
-    """Module for using filesystem (directory with files) for inventory storage"""
+class FilesystemInventory(InventoryStorage):
+    """Module for using filesystem (directory with files)
```

Comment:
Maybe make this module level docstring and vice versa?

Reply:
yeah, make sense. Thank you